package.json

```diff
@@ -6,14 +6,14 @@
   "main": "main/server.ts",
   "scripts": {
     "dev": "nodemon --exec 'ts-node --esm' src/server.ts",
```

Comment:
Can you update this dev script to take advantage of your changes to the `start` script? Should be:
```
nodemon --exec 'node --loader ts-node/esm' src/server.ts"
```

Reply:
Done